### DISCUSSION TITLE
make menuconfig has different settings choices.

### DISCUSSION THREAD

**Discussion by darkuncle33** [Desuuuu/klipper#61] [Q&A]
I can't get this to work.
The make menuconfig makes me pic usb with gpio pins or one of the uart 1 or 3 with gpio pins.
I can pic also pick dgus and uart 3 but no baud rate shows unless I pic usb.
I tried it 5 time with settings trying to get close to the guide to install and the screen will not connect. 
Neither will the pi for that matter.
I am new to klipper but do have 3 printers running it now.

**Comment by Desuuuu** [comment]
Could you share some details about what board you're using and how it's wired to your host and to the touchscreen?

A screenshot of the menu would help as well.

**Reply by darkuncle33** [reply]
Stock board. Stock wiring. I can have a screen shot this evening around 6:00 pm

**Reply by darkuncle33** [reply]
I also tried it with mainsailos as well.

**Reply by Desuuuu** [reply]
What printer are we talking about?

**Reply by darkuncle33** [reply]
Sorry, Ender 6. I was trying to fallow the "How to Install Klipper on Ender 6: Config and Setup" on 3D print Beginner. To use the stock screen.

**Reply by Desuuuu** [reply]
The menu was modified recently upstream and that tutorial wasn't updated.

From what I can tell, you should configure the firmware like this:

![image](https://user-images.githubusercontent.com/387008/123959945-458f2000-d99e-11eb-9068-b927268a7ce3.png)

**Reply by darkuncle33** [reply]
Ok I will give it a try this evening. I will take screen shots and post the results.

**Reply by darkuncle33** [reply]
Ok I tried those settings and they worked. Except The screen connects but will not work. 
I can control the printer from the web interface. Just no touchscreen support. The only touch buttons that work on the screen are the "settings" button and I can change the language to English. The other touch buttons don't work.

**Reply by Desuuuu** [reply]
There is no language setting on my touchscreen implementation.

**Reply by darkuncle33** [reply]
I meant the language button registers an touch. 
So when I turn it on the Klipper logo comes on.  Then it connects, the only touch that registers is the Settings button. When I hit settings the screen changes and in that screen the only button that works is Language and the back button.

**Reply by darkuncle33** [reply]
Also the English button registers a touch after I touch language. 
The Klipper logo show on the splash screen. Could the dwin set not have loaded correctly? Maybe load it again?

**Reply by Desuuuu** [reply]
There shouldn't be a language button.

![image](https://raw.githubusercontent.com/Desuuuu/DGUS-reloaded-Klipper/master/project/DWIN_SET/009_settings_menu.bmp)

**Reply by darkuncle33** [reply]
Ok I have one so the dwin set is not loaded correctly. Is that possible?

**Reply by darkuncle33** [reply]
I thought of reloading it but I have the Klipper splash screen. Maybe it is corrupted?

**Reply by Desuuuu** [reply]
Possibly. Follow the steps outlined here, including the optional step: https://github.com/Desuuuu/DGUS-reloaded-Klipper/wiki/Flashing-the-firmware

**Reply by darkuncle33** [reply]
I will do that. I will post the results. Thanks!

**Reply by darkuncle33** [reply]
It didn't work. The screen connects but no touch function works. I put stock firmware back on it to see if the screen was working and it worked with stock creality firmware. 
The printer will with the web interface but the screen will not work. 
I installed a TFT and klipper screen. 
Thanks for trying to help but I just can't get it to work so I am giving up.
